Dockerfile

```diff
@@ -102,5 +102,8 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
     MASS \
     GGally
 
+# Required for summarizing the RNA-seq matrices
+RUN Rscript -e "install.packages('reshape2')"
```

Comment:
This should already be on the container. As I am adding this to CI, I will also remove this and my assumption will get tested.